web/views/status.erb

```diff
@@ -24,7 +24,19 @@
         </tr>
         <tr>
           <td>status</td>
-          <td><%= @status.status %></td>
+          <td>
+            <div class="pull-left"><%= @status.status %></div>
+              <div class="progress pull-right" style="width: 90%; margin-bottom: 0;">
```

Comment:
I think that the only inline style which make sense if `width: <%= @status.pct_complete %>. All other styles should be in css.